web/src/api.js

```diff
@@ -3,6 +3,23 @@ import { compareVersions } from "./utils.js";
 import Ajv from "ajv";
 const ajv = new Ajv();
 
+export const INTERNEL_PLUGINS = {
+  BrowserFS: {
+    uri:
+      "https://gist.githubusercontent.com/oeway/68acb3e239c40ab1001c8c1bd70dba33/raw/BrowserFS.imjoy.html",
```

Comment:
It would be nice to move this source to an official plugin repo.

Reply:
Right, maybe we can have a core plugin repo to host internal plugins. Will think about it.

Reply:
I created one here: https://github.com/imjoy-team/core-plugins